pkg/scheduler/cache/cache.go

```diff
@@ -171,23 +171,21 @@ func (db *DefaultBinder) Bind(kubeClient *kubernetes.Clientset, tasks []*schedul
 			},
 			metav1.CreateOptions{}); err != nil {
 			klog.Errorf("Failed to bind pod <%v/%v> to node %s : %#v", p.Namespace, p.Name, task.NodeName, err)
-			errTasks = append(errTasks, task)
```

Comment:
maybe we can record successful bind event here and avoid lots of code change?

Reply:
A good idea! But I find that it need a instance of event recorder. So is it need to put it in DefaultBinder or pass in as a param?

Reply:
solved in latest commit.